src/modules/transaction/utils.ts

```diff
@@ -98,13 +157,18 @@ export function buildTransactionWithFromPayload(
 export type TransactionHrefOptions = {
   txHash?: string
   address?: string
+  isCrossChain?: boolean
   blockNumber?: number
 }
 
 export function getTransactionHref(
-  { txHash, address, blockNumber }: TransactionHrefOptions,
+  { txHash, address, blockNumber, isCrossChain }: TransactionHrefOptions,
   network?: number
 ) {
+  if (isCrossChain) {
+    return `https://axelarscan.io/gmp/${txHash}`
+  }
```

Comment:
should we grab this from the AxelarProvider?

Reply:
In order to avoid including the dependency here, I've changed the mechanism so we include in the transaction the cross chain provider, which is later used here to choose which URL to be written here.